SeleniumTestXUnit/Core/Drivers/Interfaces/GenericWebDriver.cs

```diff
@@ -0,0 +1,9 @@
+﻿using System;
+using OpenQA.Selenium;
+
+namespace SeleniumTest.Core.Interfaces;
+
+public interface IGenericWebDriver : IDisposable
```

Comment:
Not sure about your team code conventions but usually the file name is the same as the interface name, an "I" is missing at the beginning -> IGenericWebDriver.cs